xsuite/src/world/lsworld.ts

```diff
@@ -170,6 +170,47 @@ export class LSWorld extends World {
     return this.proxy.setPreviousBlockInfo(block);
   }
 
+  private async updateCurrentBlockInfo(
+    updateFn: (block: Required<Block>) => Required<Block>,
+  ) {
+    const networkStatus = await this.proxy.getNetworkStatus(0);
```

Comment:
Let's add a method `world.getNetworkStatus`, and use it here and in the tests.

By the time you work on the PR, I might have already added it to the dev branch already.